users/serializers.py

```diff
@@ -25,3 +26,24 @@ class Meta(UserPublicSerializer.Meta):
             "date_of_birth",
             "profile_photo",
         ]
+
+
+class SignupSerializer(serializers.Serializer):
+    username = serializers.CharField()
+    password = serializers.CharField(
+        write_only=True, required=True, validators=[validate_password]
+    )
+
+    class Meta:
+        model = User
+        fields = ("username", "password")
+
+    def create(self, validated_data):
+        user = User.objects.create(
```

Comment:
We should check if the user already exists. We can do it in the `validate` method

Reply:
Validate method was implemented